### ISSUE TITLE
Unblock blocked translator

### ISSUE THREAD

**Issue by FALCONIDAE06** [CarlosEsco/Neko#1160]
### Describe your suggested feature

I accidentally blocked a scanlator that I didn't want to, but I can't seem to find any option to remove blocking. Could you add a way to remove the blocks?

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I will fill out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
It's under mangadex site settings in the settings screen
